src/main/groovy/netflix/nebula/dependency/recommender/ExtendRecommenderConfigurationAction.java

```diff
@@ -0,0 +1,33 @@
+package netflix.nebula.dependency.recommender;
+
+import org.gradle.api.Action;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+
+import java.util.Set;
+
+public class ExtendRecommenderConfigurationAction implements Action<Configuration> {
+    private final Configuration bom;
+    private final Set<String> excludedConfigurationNames;
+    private final Project project;
+
+    public ExtendRecommenderConfigurationAction(Configuration bom, Set<String> excludedConfigurationNames, Project project) {
+        this.bom = bom;
+        this.excludedConfigurationNames = excludedConfigurationNames;
+        this.project = project;
+    }
+
+    @Override
+    public void execute(Configuration files) {
```

Comment:
Why is the parameter named files?

Reply:
Renamed to `configuration`, `files` was just the IntelliJ default parameter name